DemoInfo/DP/DemoPacketParser.cs

```diff
@@ -45,7 +48,13 @@ public static void ParsePacket(IBitStream bitstream, DemoParser demo)
 					new UpdateStringTable().Parse(bitstream, demo);
 				} else if (cmd == (int)NET_Messages.net_Tick) { //and all this other stuff
 						new NETTick().Parse(bitstream, demo);
-				} else {
+				}
+				#if !SLOW_PROTOBUF
```

Comment:
I guess these checks are no longer appropriate now that you've removed the alternate code.